.coafile

```diff
@@ -31,6 +31,9 @@ bears = SpaceConsistencyBear
 bears = GitCommitBear
 shortlog_trailing_period = False
 shortlog_regex = ([^:]*|[^:]+: [A-Z0-9*].*)
+body_close_issue = True
+body_close_issue_full_url = True
+body_close_issue_last_line = True
```

Comment:
^^Didn't quite get you. I just followed [this.](https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html)